includes/models/model.llms.order.php

```diff
@@ -317,9 +337,18 @@ private function calculate_next_payment_date( $format = 'Y-m-d H:i:s' ) {
 		if ( 0 != $end_time && ( $next_payment_time + 23 * HOUR_IN_SECONDS ) > $end_time ) {
 			$ret = '';
 		} elseif ( $next_payment_time > 0 ) {
-			$ret = date_i18n( $format, $next_payment_time );
+			$ret = date( $format, $next_payment_time );
```

Comment:
what's the reason for this change? Is this intended? Just a question.

Reply:
My thinking is that this is an internal/private function that we request a specific format for so we actually don't want any modifications to that requested format adjusted by `date_i18n()`

Reply:
I see, makes sense, thanks.